features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json

```diff
@@ -286,6 +286,7 @@
   "Shared.AppsAndKeys.ViewCurl.help": "The following cURL command shows how to generate an access token using\n                            the Password Grant type.",
   "Shared.AppsAndKeys.ViewCurl.help.in.a.similar": "In a similar manner, you can generate an access token using the Client Credentials grant type with\n                        the following cURL command.",
   "Shared.AppsAndKeys.ViewKeys.access.token": "Access Token",
+  "Shared.AppsAndKeys.ViewKeys.client.enable.client.credentials": "Enable Client-Credentials grant \n                                            type to generate test access tokens",
```

Comment:
why we need a dash in between client credentials.

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/7533/commits/4420fd45293b956567f2902e4f9e0e1ed33e202e